src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -129,30 +129,19 @@ class AddSecondaryLoginPage extends Component {
                     <View style={styles.mb6}>
                         <TextInput
                             label={this.props.translate('common.password')}
-                            value={this.state.password}
-                            onChangeText={password => this.setState({password})}
+                            inputID="password"
                             secureTextEntry
                             autoCompleteType="password"
                             textContentType="password"
-                            onSubmitEditing={this.submitForm}
+                            onSubmitEditing={this.onSubmit}
                         />
                     </View>
```

Comment:
I believe this should be removed and instead we should get the error at SECONDARY_LOGIN_FORM since that way it will be shown at the button.

Reply:
Not sure why the different lines are getting highlighted, I tried pointing out line 138 twice 😕